bokeh/core/_templates/autoload_js.js

```diff
@@ -25,96 +25,94 @@ calls it with the rendered model.
+    window._bokeh_is_loading = js_urls.length;
+    for (var i = 0; i < js_urls.length; i++) {
+      var url = js_urls[i];
+      var s = document.createElement('script');
+      s.src = url;
+      s.async = false;
+      s.onreadystatechange = s.onload = function() {
+        window._bokeh_is_loading--;
+        if (window._bokeh_is_loading === 0) {
+          console.log("Bokeh: all BokehJS libraries loaded");
+          run_callbacks()
+        }
+      };
+      s.onerror = function() {
+        console.warn("failed to load library " + url);
+      };
+      console.log("Bokeh: injecting script tag for BokehJS library: ", url);
+      document.getElementsByTagName("head")[0].appendChild(s);
     }
-    {%- endif %}
-
-    var js_urls = {{ js_urls }};
-
-    var inline_js = [
-      {%- for js in js_raw %}
-      function(Bokeh) {
-        {{ js|indent(6) }}
-      },
-      {% endfor -%}
-      function(Bokeh) {
-        {%- for url in css_urls %}
-        console.log("Bokeh: injecting CSS: {{ url }}");
-        Bokeh.embed.inject_css("{{ url }}");
-        {%- endfor %}
-        {%- for css in css_raw %}
-        console.log("Bokeh: injecting raw CSS");
-        Bokeh.embed.inject_raw_css({{ css }});
-        {%- endfor %}
-      }
-    ];
-
-    function run_inline_js() {
-      for (var i = 0; i < inline_js.length; i++) {
-        inline_js[i](window.Bokeh);
-      }
+  };
+
+  {%- if elementid -%}
+  var element = document.getElementById("{{ elementid }}");
+  if (element == null) {
+    console.log("Bokeh: ERROR: autoload.js configured with elementid '{{ elementid }}' but no matching script tag was found. ")
+    return false;
+  }
+  {%- endif %}
+
+  var js_urls = {{ js_urls }};
+
+  var inline_js = [
+    {%- for js in js_raw %}
+    function(Bokeh) {
+      {{ js|indent(6) }}
+    },
+    {% endfor -%}
+    function(Bokeh) {
+      {%- for url in css_urls %}
+      console.log("Bokeh: injecting CSS: {{ url }}");
+      Bokeh.embed.inject_css("{{ url }}");
+      {%- endfor %}
+      {%- for css in css_raw %}
+      console.log("Bokeh: injecting raw CSS");
+      Bokeh.embed.inject_raw_css({{ css }});
+      {%- endfor %}
     }
+  ];
 
-    if (window._bokeh_is_loading === 0) {
-      console.log("Bokeh: BokehJS loaded, going straight to plotting");
-      run_inline_js();
-    } else {
-      load_libs(js_urls, function() {
-        console.log("Bokeh: BokehJS plotting callback run at", now());
-        run_inline_js();
-      });
+  function run_inline_js() {
+    for (var i = 0; i < inline_js.length; i++) {
+      inline_js[i](window.Bokeh);
     }
-  }(this));
-});
+  }
+
+  if (window._bokeh_is_loading === 0) {
+    console.log("Bokeh: BokehJS loaded, going straight to plotting");
+    run_inline_js();
+  } else {
+    load_libs(js_urls, function() {
+      console.log("Bokeh: BokehJS plotting callback run at", now());
+      run_inline_js();
+    });
+  }
+}(this));
```

Comment:
I presume this is just shift left and removal of the wrapper? I just can't read this diff.